scripts/variantstore/wdl/GvsExtractCallsetPgen.wdl

```diff
@@ -0,0 +1,603 @@
+version 1.0
+
+import "GvsUtils.wdl" as Utils
+
+workflow GvsExtractCallset {
+    input {
+        Boolean go = true
+        # The name of the bigquery dataset containing the GVS data we are extracting
+        String dataset_name
+        # The project id for the specified bigquery dataset
+        String project_id
+        # The name of the callset we created using GvsPrepareRangesCallset to extract
+        String call_set_identifier
+
+        # Chromosome code corresponding to the set of contig names used by plink that matches those used by the
+        # reference (see plink chromosome codes here: https://www.cog-genomics.org/plink/2.0/data#irreg_output)
+        # ExtractCohortToPgen currently only supports codes "chrM" and "MT"
+        String pgen_chromosome_code = "chrM"
+        # Max number of alt alleles a site can have. If a site exceeds this number, it will not be written (max 254)
+        Int max_alt_alleles = 254
+        # If true, does not throw an exception for samples@sites with unsupported ploidy (codes it as missing instead)
+        Boolean lenient_ploidy_validation = false
+
+        # The project id for the bigquery dataset containing the cohort we created using GvsPrepareRangesCallset
+        String cohort_project_id = project_id
+        # The name of the bigquery dataset containing the cohort we created using GvsPrepareRangesCallset
+        String cohort_dataset_name = dataset_name
+        Boolean do_not_filter_override = false
+        Boolean control_samples = false
+        String extract_table_prefix
+        String filter_set_name
+        String query_project = project_id
+        # This is optional now since the workflow will choose an appropriate value below if this is unspecified.
+        Int? scatter_count
+        Int? extract_memory_override_gib
+        Int? disk_override
+        # If true, the output files will be named with a zero-padded interval number prefix
+        # If false, the output files will be named with a non-zero-padded interval number suffix
+        Boolean zero_pad_output_pgen_filenames = true
+
+        # set to "NONE" if all the reference data was loaded into GVS in GvsImportGenomes
```

Comment:
Which btw it is not for Echo, we dropped GQ 0

Reply:
Oh that's interesting.  Should the default value be changed/removed in that case?